includes/database/QMySqliDatabase.class.php

```diff
@@ -1,4 +1,8 @@
 <?php
+
+if (!defined('MYSQLI_ON_UPDATE_NOW_FLAG')) {
+	define ('MYSQLI_ON_UPDATE_NOW_FLAG', 8192);
```

Comment:
I think in this case its safe. There is another way to do this. phpadmin parses the line definition clause looking for the on update current_timestamp phrase. However, since its in PHP Storm, my guess is someone took a look at the MySQL source and figured this out. If they did remove it, that would be a pretty big change for mysqli, even if it was only internal. If it does break, then it would mean collision detection would not work as expected, which is generally kind of a poor way of doing it anyway, so not a big loss.